app/calendar_grid.py

```diff
@@ -0,0 +1,51 @@
+import calendar
+import datetime
+from typing import List, Tuple
+
+DAYS_OF_THE_WEEK: List[str] = ['Monday', 'Tuesday', 'Wednesday',
+                               'Thursday', 'Friday', 'Saturday', 'Sunday']
+
+
+def get_date_as_string(date: datetime) -> str:
+    return date.strftime("%A %B %b %Y").split()
+
+
+def get_next_date(date):
+    """Generate date objects from a starting given date."""
+    while True:
```

Comment:
Prefer `yield from (date + datetime.timedelta(days=i) for i in itertools.count(start=1))`